publish/scripts/installer.js

```diff
@@ -846,7 +846,7 @@ module.exports = function($logger, hookArgs) {
             });
         }
 
-        var buildType = isReleaseBuild || isProdEnv ? 'production' : 'development';
+        var buildType = (isReleaseBuild && isProdEnv) || (!isReleaseBuild && isProdEnv) ? 'production' : 'development';
 
```

Comment:
Seems logically equivalent to `var buildType = isProdEnv ? 'production' : 'development';`

Reply:
Yes you are right i'm adding some code to make it work as desired and send a new pull request